README.md

```diff
@@ -131,16 +131,33 @@ Dependencies:
 </dependency>
 ```
 
-### Usage Examples
+# Using BigBone
+For an easy introduction to using the library, I recommend that you take a look at the sample-* modules of this 
```

Comment:
This should probably be "we" or something neutral instead of "I".

Reply:
Good point. Was not sure about whether to use I or we. But I think it is better to use we in the long term. Will change it and update this PR.

Reply:
Updated.